projects/framework-photoshop/extensions/tool-configs/photoshop-document-publisher.yaml

```
@@ -22,15 +22,17 @@ engine:
       - type: plugin
         tags:
           - collector
-        plugin: document_collector
+        plugin: photoshop_document_collector
       - type: plugin
         tags:
           - validator
-        plugin: document_validator
+        options:
+          export_type: .jpg
+        plugin: photoshop_document_saved_validator
```

Remove export_type as is not used.
```suggestion
        plugin: photoshop_document_saved_validator
        ui: validator_label
```